src/ctapipe/calib/camera/tests/test_extractors.py

```diff
@@ -0,0 +1,155 @@
+"""
+Tests for StatisticsExtractor and related functions
+"""
+
+import numpy as np
+import pytest
+from astropy.table import Table
+from astropy.time import Time
+
+from ctapipe.calib.camera.extractor import PlainExtractor, SigmaClippingExtractor
+
+
+@pytest.fixture(name="test_plainextractor")
+def fixture_test_plainextractor(example_subarray):
```

Comment:
Instantiating a class is fine in the test itself, I would create a fixture for this.

Use fixtures to setup more complex test data for example.

Reply:
Okay